Projects/IntelligentObjectAvoider/README.md

```diff
@@ -10,9 +10,11 @@ The `servo` is particularly useful, because it gives the distance sensor a 120 d
 
 In order to test this script, it's advised to create at your place a small labyrinth where the `GoPiGo3` can be put in. Books, bags, stools can be used. Avoid creating very sharp corners in your little maze or too small "corridors" as your `GoPiGo3` can get stuck.
 
-This script can only run with `Python 3`.
-Don't try to run it with `Python 2` or otherwise, it will fail. Run it with the following command:
-* `python3 robot.py`
+You need to do the following things in order to be able to launch the `robot.py` script:
+1. Install `scikit-learn` package. A `pip install scikit-learn` should be enough (be mindful of whether you need to use `sudo` or not).
```

Comment:
that should be a `pip3 install scikit-learn`  if it's needed to be run with python3

Reply:
I gave there more details. Hope it's enough.